src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -176,6 +180,7 @@ const SkillsQuizStepper = ({ isStyleAutoSuggest }) => {
     };
 
     fetchLearnerCourseEnrollments();
+    // TODO: avoid disabling this rule...
```

Comment:
Great set of callouts in this component about react query and comments to ignore exhaustive dependencies 👍🏽